src/Microsoft.Azure.WebJobs.Host/app.config

```diff
@@ -1,23 +1,23 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The real change there will come in the Web.config of our Script repo. I'll make sure to do this.